files/.github/workflows/semantic-pull-request.yml

```
@@ -0,0 +1,27 @@
+name: "Semantic PR"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  main:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb # v5.0.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          # Configure which types are allowed (newline delimited).
+          types: |
+            feat
+            fix
+            chore
+            docs
+            deps
+            test
+          requre_scope: false
```

```suggestion
          requireScope: false
```

Missing `i` + [casing](https://github.com/amannn/action-semantic-pull-request/blob/b6bca70dcd3e56e896605356ce09b76f7e1e0d39/README.md?plain=1#L71)?

---

tnx